engine/src/main/java/org/camunda/bpm/engine/impl/cmd/SetExecutionVariablesCmd.java

```diff
@@ -33,6 +33,7 @@
 public class SetExecutionVariablesCmd extends AbstractSetVariableCmd {
 
   private static final long serialVersionUID = 1L;
+  protected boolean failIfNotExists = true;
```

Comment:
🙃 A new line to separate the instance from the static field
```suggestion

  protected boolean failIfNotExists = true;
```